.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: File a bug report
+labels: ["bug", "triage"]
```

Comment:
`triage` lable means that `issue needs a confirmation` or something like this. I took it from github examples, we can decide later how we will work with this.